src/applications/find-forms/components/SearchResult.jsx

```diff
@@ -70,6 +70,38 @@ const recordGAEventHelper = ({
     'search-typeahead-enabled': false, // consistently populate with false since there's no type ahead enabled for this search feature
   });
 
+const deriveRelatedTo = ({
+  vaFormAdministration,
+  formType,
+  benefitCategories,
+}) => {
+  let relatedTo = vaFormAdministration;
+
+  if (formType === 'employment') {
+    relatedTo = 'Employment or jobs at VA';
+  }
+  if (formType === 'non-va') {
+    relatedTo = 'A non-VA form. For other government agency forms, go to the ';
```

Comment:
this could be a fragment like 

```jsx
relatedTo = (
  <>
    A non-VA form. For other government agency forms, go to the <a href="https://www.gsa.gov/reference/forms">GSA forms library</a>.
  </>
)
```

Reply:
Ahhh that's how you do it

Reply:
@ncksllvn Ready to rock.